.gitmodules

```diff
@@ -1,6 +1,12 @@
 [submodule "libsbp"]
-	path = libsbp
+	path = third_party/libsbp
 	url = https://github.com/swift-nav/libsbp.git
 [submodule "libswiftnav"]
-	path = libswiftnav
+	path = third_party/libswiftnav
 	url = https://github.com/swift-nav/libswiftnav.git
+[submodule "common-cmake"]
+	path = common-cmake
```

Comment:
can we find a way keep modules in `<repo_root>/cmake`, maybe `cmake/common`? not sure how that affects imports inside of `CMakeLists.txt`'s...

Reply:
Yep that's a good idea